src/server/controllers/Ride.js

```diff
@@ -118,7 +118,7 @@ export const helperAPIQuery = (body, vendor) => {
   if (vendor === 'Lyft') {
     let helperURL = CARVIS_HELPER_API + '/lyft/getCost';
   } else {
-    let helperURL = CARVIS_HELPER_API + '/uber/requestRide';
+    let helperURL = CARVIS_HELPER_API + '/uber/getEstimate';
```

Comment:
how come this changed? we changed the name of the route? or are you pointing to a different route now than before?